content/cybersecurity/concepts/nmap/terms/tcp-connect-scan/tcp-connect-scan.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'TCP Connect Scan'
+Description: 'A type of Nmap scan that establishes a complete connection to the target.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **TCP Connect Scan** (`-sT`) is a type of Nmap scan that establishes a complete connection to the target by completing a TCP 3-way handshake. 
+Transmission Control Protocol (TCP) is a connection-oriented protocol used in modern networking, like the HTTP/HTTPS connection used to view this article.
+After Nmap's TCP connect scan completes the three way handshake, Nmap then terminates the connection.
+
+This is not to be confused with another common scan technique, TCP Syn Scan (`-sS`) which does not complete its connection to the target host. Because of this, TCP Connect Scan (`-sT`) will take longer and require more packets to perform.
+
+  The image below illustrates Nmap's TCP Connect Scan using the TCP 3-way handshake and then terminating the connection.
+  <!-- add an image here? https://static-assets.codecademy.com/skillpaths/learn-nmap/Performing_Basic_Network_Scans/network_scan_01.png -->
+  <!-- added image location docs/media/network_scan_01.png -->
+  ![Image](https://raw.githubusercontent.com/Codecademy/docs/main/media/network_scan_01.png)
```

Comment:
```suggestion
  ![TCP handshake and network scan](https://static-assets.codecademy.com/skillpaths/learn-nmap/Performing_Basic_Network_Scans/network_scan_01.png)
```

Since the image we're using is already a static asset file within Codecademy's content base, we actually don't need to save a copy and store in the `media` folder! We can just use the `static-assets` link directly and it should render fine. So let's remove the former link along with the `network_scan_01.png` file stored in `media`. 😄

Reply:
This is great to know! Thank you :smiley:

Reply:
removed `network_scan_01.png` :boom: